content/augmentor.js

```diff
@@ -0,0 +1,336 @@
+    image.className = styles.badge;
+    image.style = `height: ${height}px;`;
+
+    return image;
+  }
+
+  static getToken (token) {
+    const { TLA, balance, name } = token;
+
+    const badge = Augmentor.getBadge(token, 32);
+    // Display with 3 decimals
+    const value = parseFloat(balance).toFixed(3);
+
+    return `
+      <span title="${name}" class="${styles.token}">
+        ${badge.outerHTML}
+        <span class="${styles.balance}">
+          <span class="${styles.value}">${value}</span>
+          <span class="${styles.tla}">${TLA}</span>
+        </span>
+      </span>
+    `;
+  }
+
+  static getAccountCard (data, icon, badgesData, tokensData) {
+    const { address, name } = data;
+
+    const element = document.createElement('span');
+    element.className = styles.card;
+
+    // Get the account badges
+    const badgesHTML = badgesData.map((badgeData) => {
+      const { title } = badgeData;
+      const badge = Augmentor.getBadge(badgeData, 24);
+
+      return `
+        <span title="${title}" class="${styles.token}">
+          ${badge.outerHTML}
+          <span class="${styles.balance} ${styles['no-value']}">
+            <span class="${styles.tla}">${title}</span>
+          </span>
+        </span>
+      `;
+    }).join('');
+
+    // Get the account tokens
+    const tokensHTML = tokensData.map((token) => Augmentor.getToken(token)).join('');
+
+    const identityIcon = Augmentor.getBadge({ src: icon, title: address }, 48);
+    const nameHTML = name
+      ? `<span class="${styles.name}">${name}</span>`
+      : '';
+
+    const addressElement = document.createElement('span');
+    addressElement.className = styles.address;
+    addressElement.title = address;
+    addressElement.innerText = address;
+
+    let startedClick = false;
+    let doubleClicked = false;
+    let clickTimeout = null;
+
+    // Prevent closing on double-click on address (to select it's value)
+    addressElement.addEventListener('click', (event) => {
+      event.preventDefault();
+      event.stopPropagation();
+
+      const selectedText = window.getSelection().toString();
+
+      doubleClicked = selectedText.length > 0;
+
+      if (startedClick) {
+        startedClick = false;
+        window.clearTimeout(clickTimeout);
+        return false;
+      }
+
+      if (doubleClicked) {
+        doubleClicked = false;
+        return false;
+      }
+
+      startedClick = true;
+
+      clickTimeout = window.setTimeout(() => {
+        startedClick = false;
+        doubleClicked = false;
+        addressElement.parentElement.click();
+      }, 200);
+
+      return false;
+    });
+
+    const title = document.createElement('span');
+    title.className = styles.title;
+    title.innerHTML = nameHTML;
+    title.appendChild(addressElement);
+
+    const header = document.createElement('span');
+    header.className = styles.header;
+    header.appendChild(identityIcon);
+    header.appendChild(title);
+
+    element.innerHTML = `
```

Comment:
I think we should avoid mixing `document.createElement` and string-concat DOM creation. Perhaps it would be best to use some template library?